sql/core/src/test/resources/sql-tests/inputs/group-analytics.sql

```diff
@@ -18,12 +18,19 @@ AS courseSales(course, year, earnings);
 
 -- ROLLUP
 SELECT course, year, SUM(earnings) FROM courseSales GROUP BY ROLLUP(course, year) ORDER BY course, year;
+SELECT course, year, SUM(earnings) FROM courseSales GROUP BY ROLLUP (course, year) ORDER BY course, year;
+SELECT course, year, SUM(earnings) FROM courseSales GROUP BY ROLLUP(course, year, (course, year)) ORDER BY course, year;
+SELECT course, year, SUM(earnings) FROM courseSales GROUP BY ROLLUP(course, year, (course, year), ()) ORDER BY course, year;
 
 -- CUBE
 SELECT course, year, SUM(earnings) FROM courseSales GROUP BY CUBE(course, year) ORDER BY course, year;
+SELECT course, year, SUM(earnings) FROM courseSales GROUP BY CUBE (course, year) ORDER BY course, year;
+SELECT course, year, SUM(earnings) FROM courseSales GROUP BY CUBE(course, year, (course, year)) ORDER BY course, year;
+SELECT course, year, SUM(earnings) FROM courseSales GROUP BY CUBE(course, year, (course, year), ()) ORDER BY course, year;
```

Comment:
Could you check the behaviour of other systems (other than postgresql) when having an empty set in cube/rollup?

Reply:
> Could you check the behaviour of other systems (other than postgresql) when having an empty set in cube/rollup?

Test in Oracle, it not support empty set in groupingset of cube/rollup too.
![image](https://user-images.githubusercontent.com/46485123/100535966-681a5880-3258-11eb-84da-849266ac02fe.png)

Reply:
And which place should we prevent this illegal case?
 1. In AstBuilder throw a ParseException 
 2. In Analyzer  throw a AnalysisException
 3. in Cube/Rollup class with a assert ` assert(groupingSets.forall(_.nonEmpty))`

IMO we should throw ParseExceptionin AstBuilder

Reply:
Okay, thanks for the check. So, could you follow the PostgreSQL/Oracle behaviour? Let's revisit it if other reviewers leave comments about the behaviour.

Reply:
> Okay, thanks for the check. So, could you follow the PostgreSQL/Oracle behaviour? Let's revisit it if other reviewers leave comments about the behaviour.

Yea, updated